foxy/api/action_msgs/index-interface.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+    <title>action_msgs Message / Service / Action Documentation</title>
+    <link type="text/css" rel="stylesheet" href="../msg-styles.css" />
+  </head>
+  <body>
+    <div id="container">
+      <h1>action_msgs Message Documentation</h1>
+      <ul>
+        <li><a href="../../html/action_msgs/msg/GoalInfo.html">msg/GoalInfo</a></li>
```

Comment:
These links are broken for me. Instead of `html` I think they should be `api`. Or the generated files need be moved into an html directory.